src/components/Button/index.tsx

```diff
@@ -0,0 +1,34 @@
+import clsx from "clsx";
+import { ButtonSize, ButtonVariant, ButtonColor, ButtonProps } from "./type";
+
+export const Button = ({
+  isDisabled,
+  isLoading,
+  color = ButtonColor.Primary,
+  size = ButtonSize.Medium,
+  variant = ButtonVariant.Default,
+  content,
+  onClick,
+}: ButtonProps) => (
+  <button
+    className={clsx("btn", size, color, variant)}
+    disabled={isDisabled}
+    onClick={onClick}
```

Comment:
Chybi jeste `type` attribute - melo by to byt klasicky html type - submit, button a neco tretiho (z hlavy si nepamatuju)

Reply:
no já jenom následovala slepě ty instrukce, takže tam můžu přidat submit ale nevím co myslíš button a už vůbec to něco třetího, tak jestli mi to prosím doplníš do zadání :)